get-started/hallucination-agent/question-answer.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Question - answer 
+description: Learn how to use the kluster.ai Hallucination Detection API to validate the truthfulness of answers to questions.
+---
+
+# Question/Answer
+
+The Question/Answer method allows you to validate whether an answer to a specific question contains hallucinated information. This approach is ideal for fact-checking individual responses against provided context or general knowledge.
+
+## Prerequisites
+
+Before getting started with fine-tuning, ensure you have the following:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide
+
+
+## How it works
+
+The service evaluates the truthfulness of an answer to a question by:
+
+1. Analyzing the original question or prompt.
+2. Examining the provided answer.
+3. Determining if the answer contains hallucinated or unsupported information.
+4. Providing a detailed explanation of the reasoning behind the determination as well as the search results used for fact-checking.
```

Comment:
If this is the same text as in the other page I would move this to the overview section.